frontend/packages/core/src/chip.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { ChipProps as MuiChipProps } from "@material-ui/core";
+import { Chip as MuiChip } from "@material-ui/core";
+
+type variants = "error" | "unknown" | "pending" | "running" | "succeeded";
```

Comment:
What is the different between running and succeeded? Adding a docstring might help here

Reply:
ill change it to be more like the windshield version of WORST...BEST for now